tests/jballerina-unit-test/src/test/resources/test-src/workers/worker-cancelled.bal

```diff
@@ -1,10 +1,13 @@
+import ballerina/runtime;
+
 // Test if worker actions are panicked if the worker is cancelled before sending
 function workerCancelledBeforeSend() {
         worker wy {
             string aa = <- default;
             "message" -> default;
         }
         wy.cancel();
+        runtime:sleep(5);
```

Comment:
This test fails intermittently. 
I added this sleep following @vinok88 pointers.

Reply:
5 milliseconds is enough?

Reply:
Yes, from what @vinok88 mentioned, when there is **wait** like event, the worker gets the handle to cancel the task. 

@vinok88 please correct if i am wrong